tests/test_addon_detail.py

```diff
@@ -1,5 +1,7 @@
 import pytest
 
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common import by
```

Comment:
This doesn't seem to be used.

Reply:
Whoops! I removed the unnecessary import.